app/src/test/java/org/simple/clinic/home/patients/PatientsLogicTest.kt

```diff
@@ -118,14 +119,13 @@ class PatientsLogicTest {
     uiEvents.onNext(NewPatientClicked)
 
     // then
-    verify(uiActions).hideUserAccountStatus()
+    verify(ui, times(2)).hideUserAccountStatus()
```

Comment:
Why did we change this to `times(2)`?

Reply:
If test had the occurrence of `hideUserAccountStatus` twice, then we add times(2) to it. There are 2 scenarios where this could occur 
* Where got it button is clicked on the approval of user
* Where It couldn't satisfy any other condition of account notification